app/styles/modules/_custom-rows.scss

```diff
@@ -60,23 +60,90 @@
   color: $link-color-extra;
   font-size: 13px;
   margin-top: 16px;
-
-  .fxa-checkbox {
-    line-height: 18px;
-  }
 }
 
 .choose-what-to-sync-row,
 .checkbox-row {
   text-align: left;
+}
+
+input[type='checkbox'] {
+  appearance: none;
+  border: 1px solid rgba($grey-90, 0.3);
+  border-radius: 2px;
+  cursor: pointer;
+  display: inline-block;
+  height: 18px;
+  margin-right: 4px;
+  padding: 0 !important;
+  width: 18px;
+
+  &:not(:checked) {
+    background-color: rgba($grey-90, 0.1);
+
+    &:hover {
+      background-color: rgba($grey-90, 0.2);
+      border-color: rgba($grey-90, 0.3);
+    }
+
+    &:active {
+      background-color: rgba($grey-90, 0.3);
+    }
+
+    &:focus {
+      border: 0;
+      box-shadow: 0 0 0 1px #0a84ff inset, 0 0 0 1px #0a84ff, 0 0 0 4px rgba(10, 132, 255, 0.3);
+    }
+
+    &:focus:active {
+      background-color: rgba($grey-90, 0.3);
+    }
+
+    &:disabled {
+      background-color: rgba($grey-90, 0.1);
+    }
+  }
+
+  &:checked {
+    background-color: $blue-60;
+    background-image: url('/images/icon-check-white.svg');
```

Comment:
@shane-tomlinson there is a tiny problem (in Chrome?) where if you are loading with no cache then this will only be fetched after the checkbox is checked.
![save](https://user-images.githubusercontent.com/128755/42517408-a3f6962e-842d-11e8-946a-6648ce47d8b7.gif)

Reply:
I wonder if we can somehow prefetch the icon

Reply:
Nice catch! 

> I wonder if we can somehow prefetch the icon

We could turn it into a data uri. It'll be bigger, but one less request and load instantly.